### DISCUSSION TITLE
Try organizing test results based on the repo test folders

### DISCUSSION THREAD

**Discussion by munderseth** [testspace-com/support#3] [Ideas]
Hi @AleksIvanovSinglet, I thought you might consider organizing your test results based on the [repo source structure](https://help.testspace.com/docs/publish/push-data-results#source).  

You can give it a try and update your GitHub workflow file (`main.yml`). 

```
testspace __test/results.xml{/Source/Bogus.Tests}
```

Is this useful?

**Comment by aleks-ivanov** [comment]
This sounds quite nice, but it currently returns an error:

"Error: invalid or non-existing test-source base path "/Source/Bogus.Tests" specified!"

**Reply by munderseth** [reply]
Sorry I gave you the wrong path🤦‍♂️ . Try the following:

```
testspace __test/results.xml{Source/Bogus.Tests}
```

Note that I removed the absolute `/`.

**Reply by aleks-ivanov** [reply]
Awesome, works great and I think it is really useful :) Thank you!
